plugin-docker/che-plugin-docker-client/src/test/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfigurationTest.java

```diff
@@ -131,5 +143,98 @@ public void testDockerUriOnNonLinuxMissingProperties() throws Exception {
 
     }
 
+    /**
+     * Check that without property docker host ip from container is DEFAULT_LINUX_DOCKER_HOST_IP
```

Comment:
BTW this is funny because none of you are commenting the tests that you wrote, neither adding javadoc on interface. But you're still discussing the text on a test method. This is what I call a "double standard"